documentation/modules/proc-create-address-space-console.adoc

```diff
@@ -5,17 +5,16 @@
 [id='proc-create-address-space-console-{context}']
 = Creating an address space using the {ConsoleName}
 
-You can create a new address space using the {ConsoleName}.
+You can create a new address space using the {ConsoleName}, including configuring the endpoints of an address space using different certificate providers and creating endpoints to make the address space available for messaging applications to use.
```

Comment:
FYI if the user choses not to configure the endpoints, the system creates a default set (which are probably good for many use-cases).

Reply:
Should I add a note to this section stating that the system creates a default set if no endpoints are configured?

Reply:
Yes, I think this is a good idea.  On OpenShift, by the default will give the user openshift routes for AMQPS and AMQP WSS, secured with a system generated (self-signed) certificate and a cluster service.  On Kubernetes you get the cluster service only.

Reply:
Are the following notes sufficient?

**OpenShift note**:
If you choose not to configure the endpoints for your address space, the system creates a default set of endpoints as OpenShift routes for AMQPS and AMQP-WSS, secured with a system generated (self-signed) certificate and a cluster service.
**Kubernetes note**:
If you choose not to configure the endpoints for your address space, the system creates a default set of cluster service endpoints.

Reply:
For kubernetes, the default is a (single) cluster service.